### ISSUE TITLE
Phantom Weapon in Character Hand Colliding with nearby surfaces

### ISSUE THREAD

**Issue by jasonfogleman** [tsunamayo/Starship-EVO#4363]
### Short Description of the issue:
There appears to be a invisible phantom weapon in the character's hand as evidenced by the visible Collision Point in my characters hand. Causes near constant collisions. 

Also, Thanks for creating an amazing game. I'm so very optimistic about the Future of Starship EVO. It's miles ahead of other ship building games in terms of ease of use and the amount of fun that is baked into the building process. You're doing such a good job. 
...

### Steps to reproduce the issue:
Unsure. Possibly by deleting an item stand while an item is on it, or by moving a weapon from the hotbar into the inventory. 

...

### Environment
- Starship EVO version: early-access 21w49a
- Graphics Card: NVIDIA GeForce GTX 1060 3GB

### Additional information:
![unknown](https://user-images.githubusercontent.com/87847101/146026078-93620af8-3eae-4f7c-b345-aa9e400b47df.png)

_(Screenshots, Videos, Error logs, Related issues)_
[Player.log](https://github.com/tsunamayo/Starship-EVO/files/7712667/Player.log)
[Player-prev.log](https://github.com/tsunamayo/Starship-EVO/files/7712676/Player-prev.log)
[Double Botch.zip](https://github.com/tsunamayo/Starship-EVO/files/7712744/Double.Botch.zip)

Please also include the `Player.log` file. You can find the log file here: `%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\`

If you issue concerns a particular design please include any blueprint that could help to reproduce the issue.
Blueprint folder is over at:
`%UserProfile%\AppData\LocalLow\Moonfire Entertainment\Starship EVO\Save_Data\Blueprints\`
You will need to zip the folder to upload it. 
To do so Right click and choose [Send to -> compressed zip file] then drag the .zip onto your report to upload it.

**Comment by tsunamayo** [comment]
Hi @jasonfogleman Thanks a lot for the case, do you confirm it was following you? Collision are reactivated once the weapon is dropped.
Thanks for the cheer up mate! If you want to help (on top of booking cases of course ^^), please dont hesitate to leave a review on Steam... I got quite a back run of bad review recently, some where fair and unfortunate as people had issues with the tutorial (it should be all fixed now - it was well tested but some newcomer do weird things), but some are a bit unfair...

**Comment by jasonfogleman** [comment]
Yes I can confirm that the phantom weapon was moving with my character. It
collided as I moved through the ship.

If I equipped a new weapon then sparks would emit from the character's
hands continuously until I unequipped the new weapon. I believe the game
saw 2 weapons equipped and colliding.

I will happily leave a review. I too had an issue with the tutorial as a
recent new player and can offer you some constructive critique if that's
helpful. However judging the entire game by that would completely unfair
and I'm so pleased that I kept playing.

**Comment by tsunamayo** [comment]
@jasonfogleman Thanks a lot! I am very willing to hear any feedback from newcomer, I am very lucky to get a lot of feedback and bugs but of course it is mostly by veteran players. Unfortunately we tend to get used to all the little quirk and bugs as long as they are not game breaking...
So I have made some fix in 21w50a that could have explain that issue (I was not able to reproduce the issue in the first place). Can you tell if you reproduce?
Thanks

**Comment by jasonfogleman** [comment]
So far I have not reproduced, but I will go do questionable things with
item stands to see what I can find.
